java/junit5/src/test/java/com/saucelabs/saucebindings/junit5/examples/LocalExample.java

```diff
@@ -0,0 +1,79 @@
+package com.saucelabs.saucebindings.junit5.examples;
+
+import com.saucelabs.saucebindings.SaucePlatform;
+import com.saucelabs.saucebindings.SauceSession;
+import com.saucelabs.saucebindings.junit5.SauceBindingsExtension;
+import com.saucelabs.saucebindings.options.SauceOptions;
+import java.time.Duration;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.junit.jupiter.api.extension.TestWatcher;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+public class LocalExample {
+  WebDriver driver;
+  SauceSession session;
+
+  public static SauceOptions createSauceOptions() {
+    ChromeOptions chromeOptions = new ChromeOptions();
+    chromeOptions.addArguments("--hide-scrollbars");
+
+    return SauceOptions.chrome(chromeOptions)
+        .setPlatformName(SaucePlatform.MAC_CATALINA)
+        .setIdleTimeout(Duration.ofSeconds(30))
+        .build();
+  }
+
+  public static Boolean useSauce() {
+    System.setProperty("sauce.enabled", "true");
+    return Boolean.getBoolean("sauce.enabled");
+  }
+
+  @RegisterExtension
+  static SauceBindingsExtension sauceExtension =
+      SauceBindingsExtension.builder()
+          .capabilities(createSauceOptions())
```

Comment:
@titusfortner this line is not very intuitive `.capabilities(createSauceOptions())`. 
After seeing the example I understand what's going on. However, will someone be able to figure out how to construct the extension without an example? There are a lot of moving objects here.

Reply:
I really don't want 2 different methods here. Maybe it should be `setOptions()` since people are most likely going to be sending a browser Options instance (like `ChromeOptions`) or a `SauceOptions` instance. The code signatures themselves should be sufficient?